### ISSUE TITLE
Spyder Tutorial font too small to read

### ISSUE THREAD

**Issue by notuntoward** [spyder-ide/spyder#4554]
## Description

On my Surface Pro 4, the font is too small to read, I suppose because it's not sized according to display PPI.  There is a zoom control, but the font shrinks again as soon as I click on a link to visit a different page of the tutorial.  This is on Windows 10.

## Version and main components

* Spyder Version: 3.1.4 
* Python Version: 3.5.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pep8 >=0.6       :  1.7.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
nbconvert >=4.0  :  5.2.1 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
matplotlib >=1.0 :  2.0.2 (OK)
sympy >=0.7.3    :  1.0 (OK)
pylint >=0.25    :  1.6.4 (OK)

```

**Comment by notuntoward** [comment]
Actually, after I restarted Spyder, the tutorial fonts were somewhat bigger -- I guess it's necessary to restart after changing the High DPI preference.  Would be nice to get a notification when that control is set.

**Comment by ccordoba12** [comment]
This will be fixed in Spyder 3.2, I think. Please reopen this issue if that's not the case once we release it (in two or three weeks).
